### PR TITLE
feat(COS-2251): Add docs methods onfieldvalidity ang getformvalidity

### PR DESCRIPTION
**Pull request by brahianzapata**

### CODE REVIEW

**Review by kushki-dflasso — COMMENTED** [review]
@brahianzapata  No se va a colocar ejemplos en el README.md?

**Review by kushki-dflasso — APPROVED** [review]

File: src/repository/ICard.ts [hunk 1]
```diff
@@ -60,35 +60,49 @@ export interface ICard {
   /**
    * This event is emitted when the field validity changes
    *
-   * @function
-   * @param {(FormValidity | FieldValidity) => void} event - The function called when the form field is validited
-   * @param {FieldTypeEnum} [fieldType] - The type of form field (optional)
-   * @returns {void}
+   * @group Methods
+   * @param {(FormValidity | FieldValidity) => void} event - Callback is executed when the hosted field is validity
```
Comment (esteban-aristizabal) [inline src/repository/ICard.ts:64]: Callback is executed when the hosted field is validity => Callback is executed when the hosted field cahnges it's validity

File: src/repository/ICard.ts [hunk 2]
```diff
@@ -97,11 +111,34 @@ export interface ICard {
 
   /**
    * This function returns an {@link FormValidity} that represents the validation state of all fields
+   *
+   * @group Methods
+   *
    * @return FormValidity object with form inputs information validation
```
Comment (esteban-aristizabal) [inline src/repository/ICard.ts:117]: FormValidity object with form inputs information validation
 

 => FormValidity object with validation info of all fields

File: src/repository/ICard.ts [hunk 3]
```diff
@@ -97,11 +111,34 @@ export interface ICard {
...
    * @property {boolean} isFormValid Return validation form
    * @property {Fields} fields Object with Fields (cardholderName, cardNumber, cvv, expirationDate, deferred (optional))
+   *
    * @example
-   *  cardInstance.getFormValidity();
+   *
+   * Handling event form validity of all hosted fields
```
Comment (esteban-aristizabal) [inline src/repository/ICard.ts:123]: Handling event form validity of all hosted fields => Get field validity of all hosted fields

File: src/repository/ICard.ts [hunk 4]
```diff
@@ -97,11 +111,34 @@ export interface ICard {
...
+   *
+   * ```ts
+   * try {
+   *     cardInstance.getFormValidity((event: FormValidity) => {
+   *       // Implement your logic to handle the event FormValidity here
+   *        if (event.fields[event.triggeredBy].isValid) {
```
Comment (esteban-aristizabal) [inline src/repository/ICard.ts:129]: if (event.fields[event.triggeredBy].isValid) { => cambiar por if (event.isFormValid) {

**Review by esteban-aristizabal — CHANGES_REQUESTED** [review]

**Review by esteban-aristizabal — APPROVED** [review]

**Pull request merged**